src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/BasePreRenderer.js

```diff
@@ -0,0 +1,34 @@
+import React, {forwardRef} from 'react';
+import {ScrollView} from 'react-native-gesture-handler';
+import {View} from 'react-native';
+import _ from 'underscore';
+import htmlRendererPropTypes from '../htmlRendererPropTypes';
+
+class BasePreRenderer extends React.Component {
```

Comment:
this is still class component. :confused:

Reply:
> Looks like a perfect candidate for the functional components. Please read the style guide and correct all style issues.

@parasharrajat I interpreted your sentence as this style is very good for functional components but not good for classes.
And it was functional before and we converted it to class, so I never looked in that direction and looked at the styling guide again and againt to see where is a mistake 😃
i converted and committed Pr is ready.